ui/apps/everest/src/pages/settings/monitoring-endpoints/createEditModal/create-edit-modal.tsx

```diff
@@ -62,13 +61,14 @@ export const CreateEditEndpointModal = ({
               placeholder: Messages.fieldPlaceholders.name,
             }}
           />
-          <AutoCompleteSelectAll
-            name={EndpointFormFields.namespaces}
-            label={Messages.fieldLabels.namespaces}
+          <AutoCompleteInput
```

Comment:
Do we have changes in mocups? Where can i check them? We couldn't select all namespaces now?

Reply:
The logic changes and now we deprecated `allowedNamespaces` and can only have one namespaces per resource, because every monitoring instance and backup storage will be namespaced